examples/bri-3/src/bri/auth/agent/auth.agent.ts

```diff
@@ -1,31 +1,58 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, UnauthorizedException } from '@nestjs/common';
 import { JwtService } from '@nestjs/jwt';
+import { createJWT, ES256KSigner, hexToBytes } from 'did-jwt';
 import { ethers } from 'ethers';
-import { BpiSubject } from 'src/bri/identity/bpiSubjects/models/bpiSubject';
 import { BpiSubjectStorageAgent } from '../../../bri/identity/bpiSubjects/agents/bpiSubjectsStorage.agent';
-import { createJWT, ES256KSigner, hexToBytes } from 'did-jwt';
-import { errorMessage, jwtConstants } from '../constants';
+import { BpiSubject } from '../../../bri/identity/bpiSubjects/models/bpiSubject';
```

Comment:
I keep finding I have to redo mine as relative, whose works with absolute and how is that?